src/Makefile.am

```diff
@@ -48,6 +48,7 @@ fwts_SOURCES = main.c 				\
 	acpi/acpidump/acpidump.c 		\
 	acpi/acpiinfo/acpiinfo.c 		\
 	acpi/acpitables/acpitables.c 		\
+	sbbr/acpitables/acpitables.c 		\
```

Comment:
is it difficult to make sbbr specific changes in acpi/acpitables/acpitables.c? If there a lot of changes, that's a valid argument to make when it is sent to maintainers.